### PR TITLE
Remove TODO's

### PR DESCRIPTION
**Pull request by atobiszei**
The only ones left are related to KFS binary inputs which should be handled shortly.

JIRA:CVS-84365

### CODE REVIEW

File: src/test/model_cache_test.cpp [hunk 1]
```diff
@@ -95,7 +95,6 @@ TEST_F(ModelCacheTest, FlowTestOnlineModifications) {
     ASSERT_EQ(config.parseShapeParameter("(1,10)"), StatusCode::OK);
     ASSERT_EQ(manager->reloadModelWithVersions(config), StatusCode::OK_RELOADED);
 
-    // TODO: Uncomment once bug is resolved.
     // https://jira.devtools.intel.com/browse/CVS-68254
```
[the remark this answers is not in the document]
Comment (dkalinowski) [reply src/test/model_cache_test.cpp:98]: And bug description

File: src/test/serialization_tests.cpp [hunk 2]
```diff
@@ -81,8 +81,6 @@ const std::vector<ovms::Precision> UNSUPPORTED_OUTPUT_PRECISIONS{
     // ovms::Precision::BIN, // Cannot create tensor with such precision
     ovms::Precision::BOOL
     // ovms::Precision::CUSTOM),
-
-    // TODO: There are new API 2.0 precisions we do not support. Add tests for those.
```
Comment (dkalinowski) [inline src/test/serialization_tests.cpp:85]: We should check why `UNSUPPORTED_OUTPUT_PRECISIONS` does not contain more unsupported precisions

File: src/rest_utils.cpp [hunk 3]
```diff
@@ -24,6 +24,8 @@
 #include "tensorflow_serving/util/json_tensor.h"
 #pragma GCC diagnostic pop
 
+#include "tfs_frontend/tfstensorinfo.hpp"
```
Comment (dkalinowski) [inline src/rest_utils.cpp]: why have the file named "tfstensorinfo" if it is under tfs_frontend directory?
what is more, tensorinfo can be misleading due to already existing tensorinfo in main src directory, i would suggest the name to be precision utils?

Comment (dkalinowski) [reply src/rest_utils.cpp]: or precision conversions

File: src/test/dummy/1/mapping.json [hunk 4]
```diff
@@ -0,0 +1 @@
+{"inputs": {"b":"input_tensor"}, "outputs": {"a": "output_tensor"}}
```
Comment (dkalinowski) [inline src/test/dummy/1/mapping.json:1]: Wouldnt it break all the tests? The tests rely on a fact that default dummy has input name `b` and output name `a`

File: src/test/serialization_tests.cpp (hunk 2 above)
Comment (atobiszei) [reply src/test/serialization_tests.cpp:85]: We get:
```
unknown file: Failure
C++ exception with description "cannot locate blob for precision: UNSPECIFIED" thrown in the test body.
```
when we call Tensor constructor. The reason is that in OV 2.0 API there are no such precisions.

File: src/get_model_metadata_impl.hpp [hunk 5]
```diff
@@ -21,16 +21,17 @@
...
+#include "tensorflow_serving/apis/prediction_service.grpc.pb.h"
 #pragma GCC diagnostic pop
 
-#include "modelmanager.hpp"
 #include "status.hpp"
+#include "tensorinfo.hpp"
 
 namespace ovms {
 
-using proto_signature_map_t = google::protobuf::Map<std::string, tensorflow::TensorInfo>;
```
Comment (dkalinowski) [inline src/get_model_metadata_impl.hpp:32]: Why can't we leave it here?

**Review by dkalinowski — APPROVED** [review]

**Review by rasapala — APPROVED** [review]
LGTM

**Pull request merged**